src/test/node/users/controllers.js

```diff
@@ -92,33 +94,34 @@ test('add a user to database', t => {
     t.end(err);
   });
 });
-test('add created post using user controller', t => {
+test('add saved post using user controller', t => {
   const testBody = {
-    entity:	ENTITY.IMAGE,
-    entityKey:	'test-image-key',
-    userKey:	'test-users-key'
+    userKey:	'test-users-key',
+    itemKey:	'test-item-key'
   };
   const expected = {
     status: 200,
-    key: testBody.entityKey
+    key: testBody.itemKey
   };
   const req = httpMocks.createRequest({
     method: 'POST',
-    url: '/users/createdpost',
+    url: '/users/savedpost',
     body: testBody
   });
   const res = httpMocks.createResponse();
   clearDB().then(initMock).then(() => {
-    return new Promise((resolve) => {
-      Controller.addCreatedPost(req, res, resolve);
+    return new Promise((resolve, reject) => {
+      Controller.addSavedPost(req, res, (err, idxKey) => {
+        return (err) ? reject(err) : resolve(idxKey);
+      });
     });
   }).then(()=>{
     return new Promise((resolve, reject) => {
       const status = res.statusCode;
       const key = res._getData().data;
       t.equal(status, expected.status, 'should be same status');
       if (typeof key !== 'string' || !key.includes(testBody.userKey)) {
-        t.fail(`wrong key : ${key}`);
+        t.fail(`key is wrong: ${key}`);
```

Comment:
hmm previous one is better?
I thought it was hard to understand if we got undefined value.
`wrong key : undefined `
it doesnt sound wired?